submissions/madmaxWMFU/google-popup/css/style.css

```diff
@@ -341,16 +330,16 @@ a>.myAccount {
     padding: 5px 8px 0 16px;
 }
 
-.searchInputBox>.leftPartSearchBox {
+.search-box>.left-part-search-box {
     flex-grow: 9;
 }
 
-.searchInputBox>.rightPartSearchBox {
+.search-box>.right-part-search-box {
     display: flex;
     flex-grow: 0;
```

Comment:
Adding `margin-left: auto;` you move this block to the right side. In this case, you need to remove `flex-grow` property in the current block and in the sibling block(`.left-part-search-box`).